src/RageSoundPosMap.cpp

```diff
@@ -1,3 +1,12 @@
+/*This file is now using static_cast<int64_t>(x+0.5) instead of lrint().
```

Comment:
I don't think we needs this context. We're wholesale replacing it and if people want to compare diffs they can read the commit history.